configserver/src/main/java/com/yahoo/vespa/config/server/maintenance/ConfigServerMaintainer.java

```diff
@@ -34,7 +34,7 @@
     /** Creates a maintainer where maintainers on different nodes in this cluster run with even delay. */
     ConfigServerMaintainer(ApplicationRepository applicationRepository, Curator curator, FlagSource flagSource,
                            Instant now, Duration interval) {
-        super(null, interval, now, new JobControl(new JobControlFlags(curator, flagSource)),
+        super(null, Mode.exclusive, interval, now, new JobControl(new JobControlFlags(curator, flagSource)),
```

Comment:
Some of the configserver maintainers are shared as well. I don't remember which. This is probably fine, in any case.

Reply:
Any input on this, @hmusum?

Reply:
All config server maintainers need to run on all nodes so this looks right to me

Reply:
Perhaps it was just the ReindexingMaintainer that was shared. The others look exclusive, indeed. Just leave it like this.